src/processors/knex-processor.ts

```diff
@@ -42,15 +57,21 @@ export default class KnexProcessor<
   }
 
   async get(op: Operation): Promise<ResourceT[]> {
-    const { id, type } = op.ref;
+    const { params, ref } = op;
+    const { id, type } = ref;
     const tableName = this.typeToTableName(type);
-    const filters = op.params ? { id, ...(op.params.filter || {}) } : { id };
-    const Resource = Object.create(this.resourceFor(type));
-    const attributes = Object.keys(Resource.__proto__.attributes);
+    const filters = params ? { id, ...(params.filter || {}) } : { id };
+    const resource = Object.create(this.resourceFor(type));
+    const fields = params ? { ...params.fields } : {};
+    const attributes = getAttributes(
+      Object.keys(resource.__proto__.attributes),
```

Comment:
it seems like if we didn't use `Object.create` we could just be doing something like: 
```
const Resource = this.resourceFor(type);
// ...
Object.keys(Resource.prototype.attributes)```

Reply:
Resource is a subclass apparently, so the only why to access prototype is with `__proto__`